docs/references/express/build-mcp-service.mdx

```diff
@@ -0,0 +1,184 @@
+---
+title: Build an MCP service in your Express application with Clerk
+description: Learn how to build an MCP service using Clerk's OAuth server in your Express application.
+---
+
+If you're building an application and want to allow large language model (LLM) apps to securely access user data within your app, you'll need to set up a Model Context Protocol (MCP) service. MCP enables external tools like LLM-based apps to request user permission to access specific data from your service. Refer to the [dedicated MCP section](/docs/oauth/how-clerk-implements-oauth#building-a-model-context-protocol-mcp-service-using-clerk-s-o-auth-server) for more information.
+
+There are two parties involved in MCP:
+
+- The `client` is the LLM application that wants to access another service on a user's behalf. For example, Claude would be the "client" if it wants to get access to Gmail.
+- The `server` is the system that hosts the protected resources the client wants to access. In this example, this would be Gmail. This is sometimes referred to as the "resource server" or "MCP server".
```

Comment:
May be worth adding some sort of note about how "client" and "server" are normally used to refer to frontend and backend, and they are specifically different in this context, which can be confusing but is important to remember